core/templates/dev/head/pages/notifications-dashboard-page/notifications-dashboard-page.mainpage.html

```diff
@@ -112,5 +112,5 @@ <h2 class="oppia-page-heading">Notifications</h2>
 
 {% block footer_js %}
   {{ super() }}
-  <script src="/templates/dev/head/pages/notifications_dashboard/NotificationsDashboard.js"></script>
+  <script src="/templates/dev/head/pages/notifications-dashboard-page/notifications-dashboard-page.controller.js"></script>
```

Comment:
Is this necessary? This script is already loaded [here](https://github.com/oppia/oppia/pull/6521/files#diff-36c2f6cdb36ac04c9c191307581a241cR182).

Reply:
I think I forget to remove.
Done